sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -12,7 +12,7 @@ generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/generated
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/f74cb6084d23ad6b4f589602eabb9b98b9ba03f8/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.2/TextAnalytics.json
```

Comment:
because the one on master has a bug and this one if from https://github.com/Azure/azure-rest-api-specs/pull/10691 that fixes it.

Reply:
The service team hasn't confirmed it is a bug, and we don't know when this will be merged.
I suggest target master for this preview and keep the conversation open for the next sprint

Reply:
@maririos  Laksh confirmed it is a bug and he agreed with the PR change.